configure.ac

```diff
@@ -1477,6 +1477,20 @@ AS_VAR_IF([enable_pystats], [yes], [
   AC_DEFINE([Py_STATS], [1], [Define if you want to enable internal statistics gathering.])
 ])
 
+# Check for --enable-pytimers
+AC_MSG_CHECKING([for --enable-pytimers])
```

Comment:
Please document the new configuration option in https://docs.python.org/dev/using/configure.html doc.